media_driver/linux/common/os/mos_os_specific.c

```diff
@@ -5704,7 +5704,7 @@ MOS_STATUS Mos_Specific_WaitForBBCompleteNotifyEvent(
 {
     MOS_UNUSED(pOsInterface);
     MOS_UNUSED(GpuContext);
-    usleep(uiTimeOut);
+    usleep(1000 * uiTimeOut);
 
     return MOS_STATUS_SUCCESS;
 }
```

Comment:
Linux uses usleep and its unit is us. Need to change to ms, because we passed constant with ms unit

Reply:
Windows is ok in this function, because they use Sleep and its unit is ms.